benchmarks/innerjoin_performance.jl

```diff
@@ -0,0 +1,233 @@
+using CategoricalArrays
+using CSV
+using DataFrames
+using Dates
+using PooledArrays
+using Random
+
+function run_innerjoin_tests(warmup::Bool = false)
+    warmup || run_innerjoin_tests(true)
+    Random.seed!(1234);
+    @info warmup ? "warmup" : "testing performance"
+    df = DataFrame(llen=[], rlen=[], type=[], time=[], alloc=[], gc=[])
+    # change line below to match
+    # your preferred range of values tested
+    # and memory availability in your system
+    for llen in [10^3, 10^6], rlen in [2*10^7]
+        if warmup
+            llen, rlen = 1000, 1000
+        else
+            println("\nSize:")
+            @show llen, rlen
+        end
+        println()
+        warmup || @info "sorted string unique"
```

Comment:
Maybe add a line break before each `@info` line for readability?

Reply:
added